dexs/thalaswap-v2/index.ts

```diff
@@ -48,18 +53,26 @@ const fetch = async (timestamp: number) => {
     0
   );
 
-  const protocolFeeRatio = (await fetchURL(protocolRatioQueryURL))?.data;
-  const dailyProtocolRevenue = dailyFees * protocolFeeRatio;
-  const totalProtocolRevenue = totalFees * protocolFeeRatio;
+  const dayRevenueQuery = (await fetchURL(revenueEndpoint(timestamp, "1D")))
+    ?.data;
+  const dailyRevenue = dayRevenueQuery.reduce(
+    (partialSum: number, a: IVolumeall) => partialSum + a.value,
+    0
+  );
+
+  const totalRevenueQuery = (await fetchURL(revenueEndpoint(0, "ALL")))?.data;
+  const totalRevenue = totalRevenueQuery.reduce(
+    (partialSum: number, a: IVolumeall) => partialSum + a.value,
```

Comment:
can you add a filter here to ignore datapoints with `a.timestamp > timestamp`

this way, we avoid setting setting incorrect total revenue for past records while refilling, same check for volume